modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/components/TeamMembersTable/TeamMembersTable.js

```diff
@@ -165,7 +167,7 @@ const TeamMembersTable = ({
 			<div className="cp-team-members-table-wrapper overflow-auto">
 				{!totalUserAccounts && !(loading || searching) && (
 					<div className="d-flex justify-content-center pt-4">
-						No team members were found.
+						{i18n.translate("no-team-members-were-found")}
```

Comment:
Please, take out the double quotes
```suggestion
						{i18n.translate('no-team-members-were-found')}
```